docs/Installation.md

```diff
@@ -132,26 +132,19 @@ conda deactivate
 
 ## Developer instructions
 
-If you plan to contribute to Clinica or want to test the current development version:
-
-- Clone the Git repository
-
-    ```sh
-    git clone https://github.com/aramis-lab/clinica.git
-    cd clinica
-    git checkout dev
-    ```
+This section is intended for users who plan to contribute to Clinica or test the current development version.
```

Comment:
I would add here the instructions to create a Conda environment by using the `environment.yml` file and then install Clinica "dev" with poetry inside. Btw, what's the behavior of `poetry shell` triggered from inside a conda env?

Reply:
> I would add here the instructions to create a Conda environment by using the environment.yml file and then install Clinica "dev" with poetry inside.

Good point. Although technically, any Python 3.7 provider (system, pyenv, homebrew) would work too with these instructions.

> `Btw, what's the behavior of poetry shell triggered from inside a conda env?`

Passthrough.

Reply:
^ precisely, you get:

```console
$ poetry shell
Virtual environment already activated: /path/to/env
```